libraries/AP_ExternalAHRS/AP_ExternalAHRS_VectorNav.cpp

```diff
@@ -459,6 +464,12 @@ void AP_ExternalAHRS_VectorNav::initialize() {
         run_command("VNWRG,75,3,%u,01,0721", unsigned(imu_rate / get_rate()));
         run_command("VNWRG,76,3,%u,31,0001,0106,0613", unsigned(imu_rate / 50));
         run_command("VNWRG,77,3,%u,49,0003,26B8,0018", unsigned(imu_rate / 5));
+
+        WITH_SEMAPHORE(state.sem);
+        set_default_sensors(uint16_t(AP_ExternalAHRS::AvailableSensor::IMU) |
```

Comment:
Apologies for my ignorance how how this method works

Will this override someone's set param? Let's say they have a VN-300 connected, but don't want to use the IMU in EKF3. Do they have to set this param on every startup, or will it notice their param isn't default and leave it untouched?